### ISSUE TITLE
[Feature Request]: Add the synchronized "bookmarks.json" file

### ISSUE THREAD

**Issue by KingPollux** [remotely-save/remotely-save#624]
### What feature are you suggesting?

Dear programmer,

I really appreciate your open-source program; it's very useful. I'd like to request that you consider adding a "synchronize file .obsidian/bookmarks.json" feature to this program. This would allow users to easily sync the contents of file bookmarks .

Thank you for your efforts and contributions!

Best regards,

### What remote cloud services are you using, or suggesting adding the feature to?

webdav (坚果云 JianGuoYun/NutStore)

### Ensure no sensitive information

- [X] I ensure that no sensitive information is submitted in the issue.

**Comment by fyears** [comment]
yeah sure i never know about bookmarks.js. But I would implement the same "newer version wins" logic

**Comment by fyears** [comment]
please check out 0.5.14
